.deploy.yml

```diff
@@ -15,6 +15,10 @@ steps:
   displayName: Build
 
 - task: NuGetCommand@2
+  condition: >-
```

Comment:
The `trigger` section already restrict the branch to `v3` or `feature/*`, this is not needed